test/data.test.js

```diff
@@ -0,0 +1,77 @@
+var openDigger = require('../src/open_digger')
+const fs = require('fs');
+const assert = require('assert');
+const _ = require('lodash');
+const _isEqual = require('lodash/isEqual');
+const FILENAME = '../test/testdata.json'; 
```

Comment:
For flexibility concern, I would like to suggest another file structure.

- Put all the data to `testdata` folder.
- Use `dataKey` as the file name of each data.
- In the file, we could provide a valid JSON object with the result and also the SQL to generate the result.

With above structure, we can add lots of test data in different files and easily to check if the data is right or not as well as the corresponding SQL.

Reply:
I roughly understand, which folder is the `testdata` folder under?